src/main/java/core/basesyntax/Calculator.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+public class Calculator {
+    public static double calculate(double firstOperand, double secondOperand, char operator) {
```

Comment:
```suggestion
    public double calculate(double firstOperand, double secondOperand, char operator) {
```
it is better not to use static :) your code is much more flexible without it

Reply:
done)